### ISSUE TITLE
minor UI issue: "parser" specific options are not listed in --help

### ISSUE THREAD

**Issue by yarikoptic** [con/tributors#21]
e.g.
```
$> tributors init zenodo       
Please provide the zenodo doi with --doi
```
but
```
$> tributors init zenodo --help
usage: tributors init [-h] [--force] [--repo REPO]
                      [{zenodo,allcontrib,codemeta,all} [{zenodo,allcontrib,codemeta,all} ...]]

positional arguments:
  {zenodo,allcontrib,codemeta,all}
                        Metadata file parsers to update or initialize.

optional arguments:
  -h, --help            show this help message and exit
  --force               If exists, overwrite existing .all-contributorsrc
  --repo REPO           The repository URI, if not exported to GITHUB_REPOSITORY
```
I guess `--help` could list options for all the parsers and annotate to which any applies?

**Comment by vsoch** [comment]
If you want to address this you can take a shot, but this is intentional because they aren't actually options, they are parsed from "extra." I've created very detailed tables of options in the docs as a workaround / helper to that.

**Comment by yarikoptic** [comment]
Why not to move those tables to `--help` output or ideally reuse for both `--help` and docs (bigger undertaking, IMHO not immediately needed). FWIW, not directly related, in DataLad we came up with similarish setups for "cmdline" and "Python" examples which are listed in --help and also rendered in the docs, see e.g. https://github.com/datalad/datalad/blob/HEAD/datalad/distribution/install.py#L99

But again - it is a "minor UI issue" so let's just keep it in mind for now.

**Comment by vsoch** [comment]
Yeah, these variables are hugely un-likely to be used in the first place (mostly updating filename defaults that will never change) so I'm not going to volunteer to do this issue. But if this is important to you, feel free to take a shot / PR!
